### PR TITLE
fix: Fix docker image for feature-server

### LINKED ISSUE #3265: feast-feature-server ModuleNotFoundError: No module named 'feast.embedded_go'
## Expected Behavior 
Install [feature-server helm-chart](https://github.com/feast-dev/feast/tree/master/infra/charts/feast-feature-server), I expect to see the pod running and not have any module import errors

## Current Behavior
feature-server is unhealthy.
```
Traceback (most recent call last):
  File "/usr/local/bin/feast", line 5, in <module>
    from feast.cli import cli
  File "/usr/local/lib/python3.8/site-packages/feast/__init__.py", line 7, in <module>
    from feast.infra.offline_stores.bigquery_source import BigQuerySource
  File "/usr/local/lib/python3.8/site-packages/feast/infra/offline_stores/bigquery_source.py", line 8, in <module>
    from feast.feature_logging import LoggingDestination
  File "/usr/local/lib/python3.8/site-packages/feast/feature_logging.py", line 8, in <module>
    from feast.embedded_go.type_map import FEAST_TYPE_TO_ARROW_TYPE, PA_TIMESTAMP_TYPE
ModuleNotFoundError: No module named 'feast.embedded_go'
```

## Steps to reproduce
Use the instructions to deploy feature-server using helm: https://github.com/feast-dev/feast/tree/master/infra/charts/feast-feature-server

Create a file called feature_store.yaml with the content below:

```
project: test_feast
provider: aws
online_store:
  type: dynamodb
  region: xxx
registry:
  registry_type: sql
  path: postgresql://user:[password@x.x.x.x](mailto:password@x.x.x.x):5432/registry
offline_store: null
entity_key_serialization_version: 2
```

Run: `helm install feast-feature-server feast-charts/feast-feature-server --set feature_store_yaml_base64=$(base64 feature_store.yaml)`

### Specifications

- Version: Chart version 0.25.1
- Platform: Kubernetes version 1.22.11
- Subsystem:

## Possible Solution
Add `__init__.py` file at https://github.com/feast-dev/feast/tree/master/sdk/python/feast/embedded_go

### PR DESCRIPTION
**Pull request by kevjumba**
Signed-off-by: Kevin Zhang <kzhang@tecton.ai>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #3265

### CODE REVIEW

File: sdk/python/feast/infra/feature_servers/multicloud/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,8 @@ RUN apt update && \
 
 RUN pip install pip --upgrade
 COPY . .
```
Comment (adchia) [inline sdk/python/feast/infra/feature_servers/multicloud/Dockerfile:11]: do we still need this?

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
 
   build-publish-docker-images:
     runs-on: ubuntu-latest
-    needs: get-version
+    needs: [get-version, build_wheels]
```
Comment (adchia) [inline .github/workflows/publish.yml:49]: should be after the publish sdk step not build wheels

Comment (kevjumba) [reply .github/workflows/publish.yml:49]: Oh right fixed

Comment (kevjumba) [reply .github/workflows/publish.yml:49]: Good catch

File: sdk/python/feast/infra/feature_servers/multicloud/Dockerfile (hunk 1 above)
Comment (kevjumba) [reply sdk/python/feast/infra/feature_servers/multicloud/Dockerfile]: removed

**Review by adchia — APPROVED** [review]
/lgtm

**Pull request merged**